src/Select.js

```diff
@@ -158,9 +158,8 @@ class Select extends Component {
 	 * @inheritDoc
 	 */
 	prepareStateForRender(data) {
-		return Object.assign({}, data, {
-			items: this.items.map(item => Soy.toIncDom(item))
-		});
+		data.items = this.items.map(item => Soy.toIncDom(item));
+		return data;
```

Comment:
Was this what you were concerned about @eduardolundgren? In fact, there was no reason to use Object.assign here once we can only set a new value for any property into `data` object.